scripts/build/termux_step_create_pacman_package.sh

```diff
@@ -4,43 +4,18 @@ termux_step_create_pacman_package() {
 		rm -rf data
 	fi
 
-	local TERMUX_PKG_INSTALLSIZE
+	local TERMUX_PKG_INSTALLSIZE TERMUX_PKG_FILE TERMUX_PKG_ARCH TERMUX_PACMAN_PACKAGE_COMPRESS_CMD
+
 	TERMUX_PKG_INSTALLSIZE=$(du -bs . | cut -f 1)
 
+	# Set TERMUX_PKG_FILE, TERMUX_PKG_ARCH and TERMUX_PACMAN_PACKAGE_COMPRESS_CMD
+	termux_set_package_file_variables "$TERMUX_PKG_NAME" "false"
+	shell__validate_variable_set TERMUX_PKG_FILE termux_step_create_pacman_package " for package \"$TERMUX_PKG_NAME\"" || exit $?
+	shell__validate_variable_set TERMUX_PKG_ARCH termux_step_create_pacman_package " for package \"$TERMUX_PKG_NAME\"" || exit $?
+	shell__validate_variable_set TERMUX_PACMAN_PACKAGE_COMPRESS_CMD termux_step_create_pacman_package " for package \"$TERMUX_PKG_NAME\"" || exit $?
+
```

Comment:
`set -e` in our scripts will cause the whole script to fail with error even without these `|| exit $?`.

Reply:
Some util scripts were meant to be called where `set -e` may not be set, so likely copied from that. Can be removed.